### PR TITLE
Added tests for test application request validation

### PR DESCRIPTION
**Pull request by scholzi100**

### CODE REVIEW

File: src/test/java/org/microbean/jersey/netty/RequestTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
+package org.microbean.jersey.netty;
```
Comment (ljnelson) [inline src/test/java/org/microbean/jersey/netty/RequestTest.java:17]: Please add the required header.

File: src/test/java/org/microbean/jersey/netty/RequestTest.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import javax.ws.rs.core.Response;
+import java.net.InetSocketAddress;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class RequestTest {
+
+    private static NioEventLoopGroup group;
+    private static WebTarget webTarget;
```
Comment (ljnelson) [inline src/test/java/org/microbean/jersey/netty/RequestTest.java]: Minor formatting issue: please use two spaces throughout and separate variable declarations with a blank line.

**Review by ljnelson — CHANGES_REQUESTED** [review]
Thank you!  This looks fine; I had only a couple of tiny formatting concerns.

File: src/test/java/org/microbean/jersey/netty/RequestTest.java (hunk 2 above)
Comment (scholzi100) [reply src/test/java/org/microbean/jersey/netty/RequestTest.java]: Should I also add a PR for .editorconfig?

Comment (ljnelson) [reply src/test/java/org/microbean/jersey/netty/RequestTest.java]: Sure; hadn't heard of this file/project before.

**Review by ljnelson — APPROVED** [review]
LGTM; will merge soon

**Comment by ljnelson** [comment]
For posterity (I'll fix it because I'm knee deep in all this at the moment), the `RequestTest` that sends a `POST` to `/hork` fails properly with a `405` since the `Hork` resource does not support `POST` but only `GET`.

**Comment by ljnelson** [comment]
Changing the endpoint from `/hork` to `/hoopy` fixes this issue; I think it was just a typo in the PR.  This also now reproduces #12 so now I can dig into that; thank you!

**Pull request merged**